jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -65,7 +65,7 @@
     # Template defaults.
     jenkins_node: 'e2e'
     test-owner: 'wonderfly'
-    emails: 'wonderfly@google.com,qzheng@google.com,lakitu-dev@google.com'
+    emails: 'gci-alerts+kubekins@google.com'
```

Comment:
this isn't going to work unless the mailing list allows posting from outside google.com.